local-interchaintest/src/utils.rs

```diff
@@ -0,0 +1,161 @@
+use cosmwasm_std::Uint64;
+use localic_std::{
+    errors::LocalError,
+    modules::cosmwasm::{contract_execute, contract_query},
+    types::TransactionResponse,
+};
+use localic_utils::{utils::test_context::TestContext, NEUTRON_CHAIN_NAME};
+use log::info;
+use orbital_core::{
+    msg::{ExecuteMsg, QueryMsg},
+    orbital_domain::UncheckedOrbitalDomainConfig,
+    state::UserConfig,
+};
+
+use crate::ACC0_KEY;
+
+pub fn query_user_clearing_acc_addr_on_domain(
+    test_ctx: &TestContext,
+    orbital_core: String,
+    user_addr: &str,
+    domain: String,
+) -> Result<Option<String>, LocalError> {
+    let clearing_acc_response = contract_query(
+        test_ctx
+            .get_request_builder()
+            .get_request_builder(NEUTRON_CHAIN_NAME),
```

Comment:
typo? or why its needed?

Reply:
yeah I thought the same but it's the syntax